src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -107,24 +128,37 @@ public function createSshTunnel(array $dbConfig)
         return $dbConfig;
     }
 
-    abstract public function createConnection($params);
+    /**
+     * @param array $params
+     * @return \PDO|mixed
```

Comment:
why mixed? does it really?

Reply:
Yeah, here `mixed` is definitely necessary as not all extractors use PDO.  oracle for example uses oci